client/src/game/rules.ts

```diff
@@ -0,0 +1,11 @@
+// Index for all rules exports
```

Comment:
is the point of this so that we can just do 1 rules import and then do `rules.card.poop()` ?

Reply:
No, it's so you can import rules like
import { cardRules, variantRules } from 'rules'
And VSCode will actually autocomplete the above and suggest names as you type.